pandas/core/arrays/datetimelike.py

```diff
@@ -33,6 +33,12 @@ def _box_func(self):
         """
         raise com.AbstractMethodError(self)
 
+    def _box_values(self, values):
```

Comment:
so for future, any methods that we implement in the sub classes but not here, should be an AbstractMethodError (not specific to this comment, just generally)